service/ztests/query-describe.yaml

```diff
@@ -0,0 +1,94 @@
+script: |
+  source service.sh
+  zed create -q test1
+  zed create -q test2
+  for file in multifrom.zed agg.zed agg-no-keys.zed; do
+    echo // === $file ===
+    query="$(cat $file | jq -Rsa .)"
+    curl -H "Accept: application/json" -d "{\"query\":$query,\"head\":{\"pool\":\"test1\"}}" $ZED_LAKE/query/describe |
+      zq -J 'sources := (over sources | id := "XXX")' -
+  done
+
+
+inputs:
+  - name: service.sh
+  - name: multifrom.zed
+    data: |
+      from (
+        pool test1
+        pool test2
+      ) | put foo := "bar"
+  - name: agg.zed
+    data: |
+      count() by key1:=v1, key2
+  - name: agg-no-keys.zed
+    data: |
+      sum(this)
```

Comment:
How about a test with two channels? And one with a channel that has both an aggregation and a sort?

Reply:
Yeah probably should add these.

Reply:
@mattnibs: Add these and I'll approve.

Reply:
working on it....